JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -122,6 +112,32 @@ module Microsoft.ApplicationInsights.Context {
             }
         }
 
+        private initializeAutomaticSessionWithData(params) {
+            if (params.length > 0) {
```

Comment:
I'm not sure, I refactored this code into this function but I didn't actually edit it. This was the structure that existed before.